src/ripple/net/impl/RPCCall.cpp

```diff
@@ -1698,7 +1672,8 @@ fromCommandLine(
     const std::vector<std::string>& vCmd,
     Logs& logs)
 {
-    auto const result = rpcClient(vCmd, config, logs);
+    auto const result =
+        rpcClient(vCmd, config, logs, RPC::apiMaximumSupportedVersion);
```

Comment:
Note, this means that as soon as we bump `apiMaximumSupportedVersion`, the `tx_history` and `ledger_header` commands will stop working (as they are no longer recognized in API version 2). Ideally at this point we should remove them from `parseCommand` in this file (and also remove function `RPCParser::parseTxHistory`). This probably needs its own issue.